yammer-metrics/src/test/java/com/wavefront/integrations/metrics/WavefrontYammerHttpMetricsReporterSecondaryTest.java

```diff
@@ -1,17 +1,9 @@
 package com.wavefront.integrations.metrics;
 
-import com.google.common.collect.Lists;
 import com.wavefront.common.Pair;
 import com.wavefront.common.TaggedMetricName;
 import com.wavefront.metrics.MetricTranslator;
-import com.yammer.metrics.core.Counter;
-import com.yammer.metrics.core.Gauge;
-import com.yammer.metrics.core.Histogram;
-import com.yammer.metrics.core.Meter;
-import com.yammer.metrics.core.MetricName;
-import com.yammer.metrics.core.MetricsRegistry;
-import com.yammer.metrics.core.Timer;
-import com.yammer.metrics.core.WavefrontHistogram;
+import com.yammer.metrics.core.*;
```

Comment:
check your code style, we don't do * imports (it makes dependency analysis harder)